cli/pcluster/config/validators.py

```diff
@@ -352,6 +352,19 @@ def disable_hyperthreading_architecture_validator(param_key, param_value, pclust
     return errors, warnings
 
 
+def extra_json_validator(param_key, param_value, pcluster_config):
```

Comment:
I'm probably more in favor of removing this validator in order to be consistent with all other parameters that cannot be updated. Or what could be useful is to change the message and suggest to use the disable_hyperthreading parameter instead: `it is highly recommended to use the disable_hyperthreading parameter in order to control the hyper-threading configuration in the cluster`

Reply:
Done